uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -104,9 +184,19 @@ sub lookupNextVisitLabel {
     return $visitLabel;
 }
 
-################################################################
-########### getFileNamesfromSeriesUID  #########################
-################################################################
+
+=pod
+
+=head3 getFileNamesfromSeriesUID($seriesuid, @alltarfiles)
+
+?????????????????????????????
```

Comment:
This function extracts from the `tarchive_files` table a list of all the DICOM files with a given seriesUID. This is useful function used spcifically when running `tarchiveLoader` on a a partial subset of the complete DICOM tar, with the partial subset defined by the optional seriesUID argument (FYI @cmadjar; details here: https://github.com/aces/Loris-MRI/pull/138).